### ISSUE TITLE
version 0.2.0 from pypi doesn't work - automatic branch?

### ISSUE THREAD

**Issue by speleo3** [johnhw/pyspacenavigator#5]
[PyPI release 0.2.0](https://pypi.org/project/spacenavigator/#history) doesn't work for me, 0.1.5 works fine.

What's suspicious about this release: it's from the [automatic](https://github.com/johnhw/pyspacenavigator/tree/automatic) branch, it's not the [tagged version 0.2](https://github.com/johnhw/pyspacenavigator/releases/tag/0.2). The tagged version (master branch) works fine.

**Comment by johnhw** [comment]
suspicious is right :) I've corrected the PyPI release now. Please let me know if this works for you.

**Comment by speleo3** [comment]
Thanks for the quick release fix! 0.2.1 detects my device and receives data.

This might be out of scope for this issue, but the data received by `callback` has changed (in commit 8dead9a50385b6ee6e2237336e7c3f84eb902431 I think). `callback` previously received values **centered at 0.0** (and scaled by `device.axis_scale`), now values are **centered at 0.5**. Also the sign (direction) of some axes has changed.

`button_callback` receives `[0, 1]` on the left button, and `[0, 0]` on the right button. Prior to 1c815eb82547ef0268d15daa1367a19c73409aaa it received 1 and 2.

My device:
```
>>> print(spacenavigator._active_device.describe_connection())
SpaceNavigator connected to 3Dconnexion Space Navigator version: 1056 [serial: ]
```

**Comment by speleo3** [comment]
I've created #6 and #7 to follow up on the 0.2.1 regressions.
